tests/test_clients.py

```diff
@@ -572,16 +574,20 @@ async def test_ensure_sequential_run_early_stop(
         assert record_indices["early_stop"] != -1, "We should stop early."
 
 
+@pytest.mark.vcr
 @pytest.mark.asyncio
 async def test_crossref_retraction_status():
     async with aiohttp.ClientSession() as session:
+        retract_processor = RetractionDataPostProcessor(
+            "stub_data/stub_retractions.csv"
```

Comment:
Okay running tests locally after this PR, we are now hitting issues as this is a relative path ungrounded in the system.

Can you move this test to intake the fixture `stub_data_dir`:

```python
async def test_crossref_retraction_status(stub_data_dir: Path):
    async with aiohttp.ClientSession() as session:
        retract_processor = RetractionDataPostProcessor(
            stub_data_dir / "stub_retractions.csv"
        )
        ...
```

Also, it looks like this test creates a 163 MB cassette. That is too big. Why is this test making a fixture still, when we are passing in a retractions CSV?